src/Infrastructure/LeanCode.Firebase.FCM/ModelBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace LeanCode.Firebase.FCM;
+
+public static class ModelBuilderExtensions
+{
+    public static void ConfigurePushNotificationTokenEntity<TUserId>(
```

Comment:
We should not force users to know how long the token might be. This should be always handled by us.

Reply:
Okay, good point. I've noticed that tokens I'm getting are ~160 chars long but our column length was 512, is the length of those token documented anywhere to be possibly this long or was this an educated guess plus some safety buffer?

Reply:
That would be great, if you find it then we could leave a comment explaining this choice with a link to those docs.

Reply:
Okay, can't find that. The only "credible" source that I found is this [SO question](https://stackoverflow.com/questions/39959417/what-is-the-maximum-length-of-an-fcm-registration-id-token). The longest token that they found is now slightly more than 300 chars. I also might mistake this exact SO thread with docs now, as my previous search was a couple of years back.